src/main/java/chatbot/Alfred.java

```diff
@@ -0,0 +1,315 @@
+package chatbot;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.File;
+import java.io.FileNotFoundException;
+
+public class Alfred {
+    protected static final String LINE = "________________________________________________________________________________________";
+    protected static ArrayList<Task> list = new ArrayList<Task>();
+
+    public Alfred(String data) {
+        java.io.File file;
+        try {
+            file = new java.io.File(data);
+            this.processData(file);
+        } catch (FileNotFoundException e) {
+            this.echo("Sorry Master Bruce. I cannot find the data. I will create a new file for you.");
+            // create a new file in the same location
+            file = new java.io.File("data/alfred.txt");
```

Comment:
I think this creates a new File object but it does not actually create the alfred.txt file? The mkdir() and createNewFile() method in the File class might be what you are looking for

Reply:
Hi when I tested by deleting the alfred.txt and then running my code it works as expected.